src/utils/load-image.tsx

```diff
@@ -29,18 +29,31 @@ export function* createImageDef(world: HubsWorld, url: string, contentType: stri
     ratio,
     projection: ProjectionMode.FLAT,
     alphaMode,
+    alphaCutoff: 0.5,
     cacheKey
   };
 }
 
-export function* loadImage(world: HubsWorld, url: string, contentType: string) {
+export function* loadImage(world: HubsWorld, eid: EntityID, url: string, contentType: string) {
   const imageDef = yield* createImageDef(world, url, contentType);
 
-  return renderAsEntity(
-    world,
-    <entity
-      name="Image"
-      image={imageDef}
-    />
-  );
+  if (MediaImageLoaderData.has(eid)) {
+    const params = MediaImageLoaderData.get(eid)!;
+    if (params.projection === PROJECTION_MODE.FLAT) {
+      imageDef.projection = ProjectionMode.FLAT;
+    } else if (params.projection === PROJECTION_MODE.SPHERE_EQUIRECTANGULAR) {
+      imageDef.projection = ProjectionMode.SPHERE_EQUIRECTANGULAR;
+    }
+    if (params.alphaMode === ALPHA_MODE.OPAQUE) {
+      imageDef.alphaMode = AlphaMode.Opaque;
+    } else if (params.alphaMode === ALPHA_MODE.BLEND) {
+      imageDef.alphaMode = AlphaMode.Blend;
+    } else if ((params.alphaMode = ALPHA_MODE.MASK)) {
+    }
+    imageDef.alphaMode === AlphaMode.Mask;
```

Comment:
I guess misstyped here

```suggestion
    } else if ((params.alphaMode = ALPHA_MODE.MASK)) {
      imageDef.alphaMode === AlphaMode.Mask;
    }
```